config/config.rb

```diff
@@ -0,0 +1,69 @@
+CONFIG = {
+  data_path: "data/",
+  delimiters: {
+    pipe: "|",
+    comma: ",",
+    space: " "
+  },
+  gender_labels: {
+    male: ["M", "Male"],
+    female: ["F", "Female"]
+  },
+  input_types: [
+    {
+      delimiter: :pipe,
```

Comment:
I'm a little unclear on the intent of ```input_types``` here. Based on the README, it seems like this specifies the format of each file to be parsed, and as such the name seems a little vague. Additionally, it looks like you're going for a declarative approach with this config setup, but is it getting a little too granular to have to specify each file individually? I don't think this will scale well.